### DISCUSSION TITLE
Spanish translation

### DISCUSSION THREAD

**Discussion by rogama25** [File-New-Project/EarTrumpet#789] [Completed discussions]
Hey! Just translated most of the missing translation strings to Spanish. I'm not sure if I should translate the pdp.xml because it's on a folder that usually it's for English, and because it's mostly the same as the Store Listing file.
![image](https://user-images.githubusercontent.com/11318350/110808351-3025a700-8284-11eb-9de7-07e6767c0e3f.png)

**Comment by riverar** [comment]
@rogama25 Thanks 🎉. Let me take a look, that's a bit weird that both are in there.

**Reply by riverar** [reply]
@rogama25 OK, I deleted the `Store listing` manual resource I uploaded (was a legacy artifact). Sorry about that, can you re-translate? Thanks again in advance!

**Comment by riverar** [comment]
Moved to archive as this is now fixed.
